tsconfig.base.json

```diff
@@ -1,16 +1,11 @@
 {
-  "extends": "@aligent/ts-code-standards/tsconfigs-base",
-  "compileOnSave": false,
-  "compilerOptions": {
-    "exactOptionalPropertyTypes": false,
-    "noUncheckedSideEffectImports": true,
-    "types": [
-      "vitest/globals",
-      "vitest/importMeta",
-      "vite/client",
-      "node",
-      "vitest"
-    ]
-  },
-  "exclude": ["node_modules", "tmp"]
+    "extends": "@aligent/ts-code-standards/tsconfigs-base",
+    "compileOnSave": false,
+    "compilerOptions": {
+        "baseUrl": ".",
+        "exactOptionalPropertyTypes": false,
+        "noUncheckedSideEffectImports": true,
+        "types": ["vitest/globals", "vitest/importMeta", "vite/client", "node", "vitest"]
```

Comment:
Did we increase our max line length?

Reply:
This is a side effect of using our `@aligent/ts-code-standard@3.0.0` package. I think we would like to re-enforce our max line length; we should make the change there.

Reply:
Line length is super arbitrary IMO, 100 is just as good as 80. Let's have the same setting across the company :+1: